Junk4Dollars-app/Utils/Time.swift

```diff
@@ -0,0 +1,46 @@
+import Foundation
+
+public struct Time {
+    var utc: String
+
+    public init(utc: String) {
+        self.utc = utc
+    }
+
+    public func local() -> String? {
```

Comment:
Can this function be deleted? In theory, everything could be refactored to use the two new functions.